app/auth/view.py

```diff
@@ -0,0 +1,119 @@
+from . import auth
+from .. import db
+from..email import send_email
+from ..model import User
+from .form import SignInForm, SignUpForm, PasswordResetForm, ChangePasswordForm, PasswordResetRequest, ChangeEmailForm
+from flask.ext.login import login_user, logout_user, current_user, login_required
+from flask import render_template, redirect, url_for, request, flash, session
```

Comment:
imports are supposed to be listed in alphabetical order, separated in block such that external dependancies are listed first then those from your application i.e http://stackoverflow.com/q/22722976